java/core/src/test/org/apache/orc/TestConf.java

```diff
@@ -30,10 +30,14 @@ public interface TestConf {
   @BeforeEach
   default void clear() {
     conf.clear();
+    conf.set("fs.defaultFS", "file:///");
+    conf.set("fs.file.impl.disable.cache", "true");
```

Comment:
These two configurations are required explicitly to avoid `Exception` for `core` module.